.github/workflows/ci.yml

```diff
@@ -35,23 +35,19 @@ jobs:
           python-version: 3.7
           with-isal: true
     steps:
-    - name: Install pigz
+    - name: Install pigz and pbzip2 MacOS
+      if: startsWith(matrix.os, 'macos')
       run: >
-          if [[ ${{ startsWith(matrix.os, 'macos') }} = true ]]; then
             brew install pigz;
```

Comment:
Can this be written as `brew install pigz pbzip2` on one line?

Reply:
I made the commit just now.